src/DoctrineServiceProvider.php

```diff
@@ -88,6 +88,7 @@ protected function ensureValidatorIsUsable()
         if (!$this->isLumen()) {
             return;
         }
+        assert(property_exists($this->app, 'availableBindings'));
```

Comment:
"added assert for lumen"

What does this mean?

Reply:
`$this->app` has "availableBindings" only when it is used in lumen.
Static analysis failed for `Illuminate\Contracts\Foundation\Application` does  not have "availableBindings" property:
`Access to an undefined property                                     Illuminate\Contracts\Foundation\Application::$availableBindings. `
https://github.com/laravel-doctrine/orm/actions/runs/8966821073/job/24623101202

The method `DoctrineServiceProvider::ensureValidatorIsUsable()` is for lumen(as shown in line 88), so the assert should be always correct here.